src/utils.jl

```diff
@@ -0,0 +1,9 @@
+elunit(x) = typeunit(eltype(x))
+typeunit(::Type) = NoUnits
+typeunit(::Type{Q}) where {Q<:Quantity} = unit(Q)
+
+uadjust(::Units, x) = x
+function uadjust(U::AffineUnits, x)
+  A = absoluteunit(U)
+  uconvert.(A, x)
+end
```

Comment:
we need another method uadjust(x) = uadjust(elunit(x), x) to be used everywhere. we don't need to store the unit in IDW nor in LWR, only in Kriging. But we can call elunit at the very end near the sigma instead.